conf/db/migration/default/V2__create_candidates_table.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE credentials
+(
+    id    uuid primary key,
+    key   varchar(32),
+    token varchar(64),
+    owner varchar(100) UNIQUE
+);
+
+
+CREATE TABLE candidates
+(
+    credential_id uuid REFERENCES credentials (id),
```

Comment:
Nice, didn't know that was possible!

Reply:
It is Postgres's way of setting up a foreign key to another table. This is why I chose the UUID.